src/cc/bpf_module.cc

```diff
@@ -95,6 +98,8 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags)
     : flags_(flags), ctx_(new LLVMContext) {
+  InitializeNativeTarget();
```

Comment:
I see. I think
b.update_table("stats", "2", "{ 2 3 0x1000000004 { 5 6 }}")
is getting too fancy. Using native sscanf for the string to get it into binary blob to send via bpf syscall... I don't see much advantage of the string vs creating the blob manually. I thought this is to help reading? then it would make a lot more sense.